<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CPhasesEditorClass</class>
  <widget class="QWidget" name="CPhasesEditorClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>255</width>
        <height>266</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Phases editor</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
          <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
          </attribute>
          <column>
            <property name="text">
              <string>Phase name</string>
            </property>
          </column>
          <column>
            <property name="text">
              <string>Aggregation state</string>
            </property>
          </column>
          <property name="toolTip">
            <string>Active phases</string>
          </property>
          
          <property name="whatsThis">
            <string>List of active phases in the flowsheet with their names and aggregation states</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QPushButton" name="pushButtonAdd">
              <property name="text">
                <string>Add phase</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
              <property name="toolTip">
                <string>Add new active phase</string>
              </property>
              
              <property name="whatsThis">
                <string>Add new active phase to the flowsheet</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButtonRemove">
              <property name="text">
                <string>Remove phase</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
              <property name="toolTip">
                <string>Remove active phase</string>
              </property>
              
              <property name="whatsThis">
                <string>Remove the selected phase from the list of active phases of the flowsheet</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
            <widget class="QPushButton" name="pushButtonOk">
              <property name="text">
                <string>OK</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButtonCancel">
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="autoDefault">
                <bool>false</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>tableWidget</tabstop>
    <tabstop>pushButtonAdd</tabstop>
    <tabstop>pushButtonRemove</tabstop>
    <tabstop>pushButtonOk</tabstop>
    <tabstop>pushButtonCancel</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>